src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,39 @@
+package step2
+
+object Calculator {
+
+    fun calculate(input: String?): Double {
+
+        val inputList: List<String> = splitInput(input)
+
+        var result: Double = inputList[0].toDouble()
+
+        var i = 1
+        while (i < inputList.size) {
+            result = computeByOperator(inputList[i], result, inputList[i + 1].toDouble())
+            i += 2
+        }
```

Comment:
```suggestion
        for (i: Int in 1 until inputList.size step 2) {
            result = computeByOperator(inputList[i], result, inputList[i + 1].toDouble())
        }
```
이렇게도 나타낼 수 있어요! 
`until`과 `step` 함수로 들어가서 확장함수, `infix` 함수 등에 대해서 좀 더 깊게 공부해보는 것도 좋을 것 같습니다!